app/src/main/java/com/github/h3lp3rs/h3lp/HelpPageActivity.kt

```diff
@@ -0,0 +1,139 @@
+package com.github.h3lp3rs.h3lp
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.github.h3lp3rs.h3lp.databinding.ActivityHelpPageBinding
+import com.github.h3lp3rs.h3lp.locationmanager.GeneralLocationManager
+import com.github.h3lp3rs.h3lp.util.GDurationJSONParser
+import com.google.android.gms.maps.MapsInitializer
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.MainScope
+
+const val EXTRA_HELP_REQUIRED_PARAMETERS = "help_page_required_key"
+const val EXTRA_DESTINATION_LAT = "help_page_destination_lat"
+const val EXTRA_DESTINATION_LONG = "help_page_destination_long"
+
+/**
+ * Activity used to display information about a person in need, their location, the path to them,
+ * the time to get there, and what help they need
+ * The user can then accept to help them (or not)
+ */
+class HelpPageActivity : AppCompatActivity(), CoroutineScope by MainScope() {
+    private lateinit var binding: ActivityHelpPageBinding
+
+    //TODO : currently, the destination is hardcoded, this will change with the task allowing
+    // nearby helpers to go and help people in need (in which case the destination will be the
+    // location of the user in need)
+    private var destinationLat = 46.519
+    private var destinationLong = 6.667
+    private var currentLong: Double = 0.0
+    private var currentLat: Double = 0.0
+
+    // TODO : this is only for displaying purposes, helpRequired will be initialized to null when
+    //  we use this activity to retrieve actual helping requests
```

Comment:
```suggestion
    // we use this activity to retrieve actual helping requests
```

Reply:
this is so that the TODO comment works properly in android studio :)